<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="redLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="greenLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="redEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="greenEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="blueEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="colorView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="randomButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QDial" name="redDial">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="blueDial">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="greenDial">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>redEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sliderUpdate()</slot>
  <slot>editUpdate()</slot>
  <slot>buttonClicked()</slot>
 </slots>
</ui>
